<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchmarkDialog</class>
 <widget class="QDialog" name="BenchmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startRead_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Read Benchmark</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="benchmark_progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCustomPlot" name="benchmark_plot" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startWrite_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Write Benchmark</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minSize_spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>73</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="repetitions_label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Repetitions</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxSize_spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>73</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>450</number>
   </property>
   <property name="value">
    <number>410</number>
   </property>
  </widget>
  <widget class="QLabel" name="maxSize_label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>115</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Size [kB]</string>
   </property>
  </widget>
  <widget class="QLabel" name="minsize_label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Size [kB]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="steps_spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>73</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="repetitions_spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>73</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="steps_label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>35</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
